ios/testmanagerd/xcuitestrunner.go

```diff
@@ -256,14 +256,12 @@ func StartXCTestWithConfig(ctx context.Context, xctestrunFilePath string, device
 			fmt.Errorf("error parsing xctestrun file: %w", err),
 		}
 	}
+	svc, _ := installationproxy.New(device)
+	installedApps, _ := svc.BrowseUserApps()
```

Comment:
are the errors ignored intentionally here?
Also, `svc` needs to be closed after we're done